src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,47 @@
-export const Tabs = () => {};
+import React from 'react';
+
+interface TabsType {
+  id: string;
+  title: string;
+  content: string;
+}
+
+type Props = {
+  tabs: TabsType[],
+  selectedTabsId: string,
+  onTabSelected: React.MouseEventHandler<HTMLAnchorElement>,
+  selectedTab: TabsType | undefined,
+};
+
+export const Tabs: React.FC<Props> = React.memo(({
+  tabs,
+  selectedTabsId,
+  onTabSelected,
+  selectedTab,
```

Comment:
Remove the `selectedTab` prop from the destructuring assignment as it is not needed.